lib/bmp280/calibration.ex

```diff
@@ -201,4 +201,55 @@ defmodule BMP280.Calibration do
   end
 
   def raw_to_humidity(_cal, _temp, _raw), do: :unknown
+
+  @gas_range_lookup1 [
```

Comment:
If you change these lookup tables to tuples, it will be a little faster. Lists are O(n) for finding the element. Tuples are O(1).